CONTRIBUTING.md

```diff
@@ -13,6 +13,7 @@ $ brew install automake ccache cmake coreutils gnu-sed go icu4c libiconv libtool
 ```
 
 ```bash#Ubuntu/Debian
+$ sudo apt update
```

Comment:
I'd add a `#` comment to this line to explain, but `zsh` at one point defaulted (and maybe still does?) to *running code from intended comments*.